src/utils/MediaQueries.js

```diff
@@ -1,10 +1,12 @@
 const size = {
   mobileS: '360px',
+  onlyMobileS: '1023px',
```

Comment:
You shouldn't make this size because we already have the same size - for example "laptop - 1"

Reply:
onlyMobileS: `(max-width: calc(${size.laptop}px - 0.02px))`,

Reply:
It's true. However, without calc will be better)